signal/tflm_signal.bzl

```diff
@@ -0,0 +1,88 @@
+#load("@tflm_pip_deps//:requirements.bzl", "requirement")
+#load("@tensorflow_cc_deps//:tensorflow.bzl", "tf_gen_op_wrapper_py")
+
+def py_tflm_signal_library(
+        name,
+        srcs = [],
+        deps = [],
+        visibility = None,
+        cc_op_defs = [],
+        cc_op_kernels = []):
+    """Creates build rules for signal ops as shared libraries.
+
+    Defines three targets:
+    <name>
+        Python library that exposes all ops defined in `cc_op_defs` and `py_srcs`.
+    <name>_cc
+        C++ library that registers any c++ ops in `cc_op_defs`, and includes the
+        kernels from `cc_op_kernels`.
+    python/ops/_<name>.so
+        Shared library exposing the <name>_cc library.
+    Args:
+      name: The name for the python library target build by this rule.
+      srcs: Python source files for the Python library.
+      deps: Dependencies for the Python library.
+      visibility: Visibility for the Python library.
+      cc_op_defs: A list of c++ src files containing REGISTER_OP definitions.
+      cc_op_kernels: A list of c++ targets containing kernels that are used
+          by the Python library.
+    """
+    binary_path = "python/ops"
```

Comment:
either update for the new location of the python code.

Reply:
Done.